<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>511</width>
      <height>471</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Code</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Group</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Quantity</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="open_button">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>500</y>
      <width>88</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>500</y>
      <width>88</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addGroup_textEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>500</y>
      <width>88</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add group</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="newGroup_textEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>499</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="group_comboBox">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>76</y>
      <width>123</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="code_textEdit">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>178</y>
      <width>123</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>178</y>
      <width>35</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>76</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Group</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="price_textEdit">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>230</y>
      <width>123</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>125</y>
      <width>60</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>23</y>
      <width>40</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="name_textEdit">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>23</y>
      <width>123</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="quantity_textEdit">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>125</y>
      <width>123</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>34</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="less_then_textEdit">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>452</y>
      <width>123</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="find_with_textEdit">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>410</y>
      <width>123</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="find_with_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Find with</string>
    </property>
   </widget>
   <widget class="QPushButton" name="less_then_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>452</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Less then</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sort_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="endEdit_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>End edit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
    <!//I've explained it all earlier>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
